server/services/errorMessageTranslator.ts

```diff
@@ -33,6 +33,8 @@ export default function translateErrorToText(
       return [`Remand periods must have a from and to date.`]
     case 'REMAND_OVERLAPS_WITH_REMAND':
       return [`Remand time can only be added once, it can cannot overlap with other remand dates.`]
+    case 'REMAND_OVERLAPS_WITH_REMAND_DETAILED':
```

Comment:
REMAND_OVERLAPS_WITH_REMAND_DETAILED generates the same message as REMAND_OVERLAPS_WITH_REMAND, but in the API they generate different messages, hence was split out into two separate enums. Will revisit, I suspect one can be removed